### PR TITLE
Amend the StorageClass page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/en/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -225,6 +225,10 @@ class. Different parameters may be accepted depending on the `provisioner`. For
 `iopsPerGB` are specific to EBS. When a parameter is omitted, some default is
 used.
 
+There can be at most 512 parameters defined for a StorageClass.
+The total length of the parameters object including its keys and values cannot
+exceed 256 KB.
```
Comment (sftim) [inline content/en/docs/concepts/storage/storage-classes.md]: Is this actually 256 KiB?
```suggestion
exceed 256 KiB.
```
?

**Comment by bradtopol** [comment]
Assigning @sftim since he is already engaged on this one.
/assign @sftim

File: content/en/docs/concepts/storage/storage-classes.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/concepts/storage/storage-classes.md]: KiB is it.

**Comment by sftim** [comment]
/sig storage

@tengqm [IMO] this either needs tech review, or I can LGTM it if I have a pointer to why those numbers are right.

**Comment by tengqm** [comment]
Parameter number limit: 
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/storage/validation/validation.go#L97
where `maxProvisionerParameterLen` is defined as 512 at
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/storage/validation/validation.go#L39

Parameter size limit:
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/storage/validation/validation.go#L109
where `maxProvisionerParameterSize` is defined at
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/storage/validation/validation.go#L38

**Pull request merged**